contributors/design-proposals/node/troubleshoot-running-pods.md

```diff
@@ -437,14 +453,15 @@ Containers unless the sandbox is restarted.
 
 ##### Step 3: kubelet API changes
 
-The kubelet exposes the new functionality in its existing `/exec/` endpoint.
+The kubelet exposes the new functionality in a new `/podDebug/` endpoint.
```

Comment:
needs updating to indicate it is given the v1.Container and optional target container name